scripts/globals/items/nomad_cap.lua

```diff
@@ -1,29 +1,29 @@
 -----------------------------------
---  ID: 16119
---  Nomad Cap
---  Transports the user to their Home Nation
+-- ID: 16119
+-- Nomad Cap
+-- Transports the user to their Home Nation
+-- TODO: Confirm wiki claims of random zone destinations among a same nation.
+-----------------------------------
+require("scripts/globals/zone")
 -----------------------------------
 local itemObject = {}
 
+local pos =
+{
+    -- Content  [Nation] = {   x,   y,   z, rot, zone},
```

Comment:
CI even looks in the comments! :D

```
Table closed without an appropriate preceding space or newline: scripts/globals/items/nomad_cap.lua:13:54
-- Content  [Nation] = {   x,   y,   z, rot, zone},                              <-- HERE
```

Reply:
I have comments ignored for things in a separate branch I've been working on, if we want all comments ignored, can bump that up in the list of checks

Reply:
In this specific case, I think its useful. But in general I'm sure we'll hit lots of false positives